rancher2/schema_app_v2.go

```diff
@@ -167,3 +134,27 @@ func appV2Fields() map[string]*schema.Schema {
 
 	return s
 }
+
+func validateAppSchema(val interface{}, key string) (warns []string, errs []error) {
+	v, ok := val.(string)
+	if !ok || len(v) == 0 {
+		return
+	}
+	_, err := ghodssyamlToMapInterface(v)
+	if err != nil {
+		errs = append(errs, fmt.Errorf("%q must be in yaml format, error: %v", key, err))
```

Comment:
Add `[ERROR]` to fmt.Errorf object strs so they show up more easily in the logs

Reply:
Updated!